src/components/Details.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { useParams } from 'react-router-dom';
+
+const Details = () => {
+  const character = useParams();
+  const details = useSelector((state) => state.characters);
+  const charDetails = details.filter((char) => +char.id === +character.id);
+
+  return (
+    <div className="details-container">
+      {charDetails.map((char, index) => (
```

Comment:
**[REQUIRED CHANGES]**

- Your app is working fine but a user can't actually search for a particular Disney as this is part of the project requirement. There's no input or search box in your design where users can search. Please, kinly add this missing feature which is key in your app functionality. see the image below:


![SEARCH](https://user-images.githubusercontent.com/59397159/176909273-c04f7a93-8597-4430-8817-9d06339cd217.png)